types/eslint/index.d.ts

```diff
@@ -946,7 +1092,21 @@ export namespace Linter {
             | 2024
             | "latest"
             | undefined;
+
+        /**
+         * Set to "script" (default) or "module" if your code is in ECMAScript modules.
+         *
+         * @default 'script'
+         *
+         * @see https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-parser-options
+         */
         sourceType?: "script" | "module" | undefined;
+        /**
```

Comment:
Sometimes there are blank lines above docs, sometimes not
Would you consider to make it consistent?

Reply:
I tried to make it consistent, but the style of property definitions varies greatly. So I only checked those with jsdocs.